README.md

```diff
@@ -47,7 +46,9 @@ You may first need to generate a CircleCI API Token from the [Personal API Token
 $ circleci setup
 ```
 
-If you are using this tool on `.com`. accept the provided default `CircleCI API End Point`. If you are using it on Server, change the value to your Server address (i.e. `circleci.my-org.com`).
+If you are using this tool on `.com`. accept the provided default `CircleCI Host`.
+
+Server users will have to change the default value to your custom address (i.e. `circleci.my-org.com`).
```

Comment:
This CLI will not actually work very well against the current production server, which creates some interesting challenges.